packages/client/src/components/uppy/uppy.ts

```diff
@@ -0,0 +1,257 @@
+import Uppy, { UppyFile } from '@uppy/core'
+import AwsS3Multipart, { AwsS3Part } from '@uppy/aws-s3-multipart'
+import {
+  AbortMultipartUploadDocument,
+  AbortMultipartUploadMutationVariables,
+  CompleteMultipartUploadDocument,
+  CompleteMultipartUploadMutationVariables,
+  CompleteMultipartUploadReturn,
+  PrepareUploadPartsDocument,
+  PrepareUploadPartsQueryVariables,
+  GetUrlReturn,
+  CreateMultipartUploadDocument,
+  CreateMultipartUploadReturn,
+  ListPartsDocument,
+  ListPartsQueryVariables,
+  useDownloadGetUrlQuery,
+  useDownloadGetUrlsQuery,
+  GetUrlsReturn,
+} from './graphql'
+import { createRequest } from 'urql'
+import type { Client } from 'urql'
+
+const uploadedParts: { [x: string]: any[] } = {}
+
+function getUploadedParts(uploadId: string) {
+  const _uploadedParts = uploadedParts[uploadId] || []
+  uploadedParts[uploadId] = _uploadedParts
+  return _uploadedParts
+}
+
+;(window as any)['uploadedParts'] = uploadedParts
+
+export function getDownloadUrls(fileKeys: string[]) {
+  const [result] = useDownloadGetUrlsQuery({
+    variables: {
+      fileKeys: fileKeys,
+    },
+    pause: !fileKeys,
+  })
+  if (fileKeys && result.data) {
+    const { downloadGetUrls } = result.data as {
+      downloadGetUrls: GetUrlsReturn
+    }
+    if (!downloadGetUrls || !downloadGetUrls.urls)
+      throw Error(result.error ? result.error.message : 'No download url found')
+    const { urls } = downloadGetUrls
+    return urls
+  }
+}
+
+export function getDownloadUrl(fileKey: string) {
+  const [result] = useDownloadGetUrlQuery({
+    variables: {
+      fileKey: fileKey,
+    },
+    pause: !fileKey,
+  })
+  if (fileKey && result.data) {
+    const { downloadGetUrl } = result.data as { downloadGetUrl: GetUrlReturn }
+    if (!downloadGetUrl || !downloadGetUrl.url)
+      throw Error(result.error ? result.error.message : 'No download url found')
+    const { url } = downloadGetUrl
+    return url
+  }
+}
+
+export function getUppy(urqlClient: Client, allowedFileTypes: string[] | null) {
```

Comment:
Please, come up with a solution how tu support bot urql and appollo clients.

Maybe interface with `executeQuery()` and `executeMutation()`